dronegen/windows.go

```diff
@@ -23,6 +23,9 @@ const (
 	toolchainDir      = `/toolchains`
 	teleportSrc       = `/go/src/github.com/gravitational/teleport`
 	webappsSrc        = `/go/src/github.com/gravitational/webapps`
+
+	relcliURL    = `https://cdn.teleport.dev/relcli-v1.1.70-beta.2-windows.exe`
```

Comment:
Final v1.1.70 will likely get cut this Thursday. I plan to bump the version with the upcoming PR that will also switch to using the production releases instance. No further relcli changes are expected.